src/util.js

```diff
@@ -14,7 +14,7 @@
  */
 var Util = window.Util || {};
 
-Util.MIN_TIMESTEP = 0.001;
+Util.MIN_TIMESTEP = 0.0001;
```

Comment:
It looks like there are different rates that can be set for reading out Android sensor data at: NORMAL, UI, GAME or FASTEST.   Firefox devicemotion events are set to use _Fastest_.

For my device, the _Fastest_ rate is about 120hz, which equals to 9ms.   So what I am getting back seems about right for Firefox.  

Maybe Chrome uses _Game_ rate?   Which on my device is ~50hz, which is 20ms.   Seems a bit off to me, but maybe still within the range?